ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Overview/WorkloadCvesOverviewPage.tsx

```diff
@@ -90,21 +67,21 @@ function WorkloadCvesOverviewPage() {
                         <CardBody>
                             {activeEntityTabKey === 'CVE' && (
                                 <CVEsTableContainer
-                                    defaultFilters={storedValue.preferences.defaultFilters}
+                                    defaultFilters={emptyStorage.preferences.defaultFilters}
```

Comment:
This prevents having to mess with the props that the containers accept, as well as prevents messing up people who already have some value in localStorage for this.